freqtrade/strategy/parameters.py

```diff
@@ -24,6 +24,45 @@
 logger = logging.getLogger(__name__)
 
 
+class DimensionProtocol(Protocol):
+    name: str
+
+
+class ft_CategoricalDistribution(CategoricalDistribution):
+    def __init__(
+        self,
+        name: str,
+        categories: Sequence[Any],
+        **kwargs,
+    ):
+        self.name = name
+        return super().__init__(categories)
+
+
+class ft_IntDistribution(IntDistribution):
```

Comment:
thinking out loud
if we name these `Integer` (and corresponding other skopt terms) - and move "name" to the back of the params - it'd be a drop-in replacement for skopt-spaces.
With clever import replacements (the documentation suggests importing them from `freqtrade.optimize.space`), it should be a (mostly) compatible replacement for skopt parameters.
alternatively, we can also alias them (Nothng speaks against having both Integer and ft_IntDistribution reference the same type).

Any objections to that?

Reply:
ok